src/stravalib/model.py

```diff
@@ -871,14 +865,17 @@ def full_photos(self) -> BatchedResultsIterator[ActivityPhoto]:
 class SummaryActivity(MetaActivity, strava_model.SummaryActivity):
     # field overrides from superclass for type extensions:
     athlete: Optional[MetaAthlete] = None
-    start_latlng: Optional[LatLon] = (
-        None  # Do we still need these overrides for latlon?
-    )
+    # These force validator to run on lat/lon
+    start_latlng: Optional[LatLon] = None
     end_latlng: Optional[LatLon] = None
     map: Optional[Map] = None
     type: Optional[RelaxedActivityType] = None
     sport_type: Optional[RelaxedSportType] = None
 
+    _latlng_check = field_validator(
```

Comment:
this is what was breaking everything. i totally missed that the attrs weren't defined in the other class but here (had been moved) thanks again @jsamoocha ✨